crates/ruma-common/src/events/message.rs

```diff
@@ -177,9 +178,15 @@ impl MessageContent {
     }
 }
 
-impl From<Vec<Text>> for MessageContent {
-    fn from(variants: Vec<Text>) -> Self {
-        Self(variants)
+impl TryFrom<Vec<Text>> for MessageContent {
+    type Error = TryFromExtensibleError;
```

Comment:
Can we have a dedicated `#[non_exhaustive] struct EmptyMessageContentError;` or sth. like that? Or do we actually want to bubble up this error somewhere?

It would probably also make sense to have this as an inherent method that returns `Option`.

Reply:
Sure we can